README.md

```diff
@@ -236,98 +240,72 @@ brew update; brew install kubernetes-helm
 
 ```bash
 helm init
+kubectl create clusterrolebinding tiller-clusterrole-binding --clusterrole=cluster-admin --serviceaccount=kube-system:tiller
```

Comment:
duplicate of line 28?

Reply:
Yes, it is a duplicate but we wanted to do this because you need to do the same thing both on AWS and locally as far as I know, right @svidrascu ?

Reply:
Yes, we want to show the same instructions in both places.